apps/daimo-mobile/src/i18n/languages/es.ts

```diff
@@ -0,0 +1,791 @@
+import { LanguageDefinition } from "./en";
+
+export const es: LanguageDefinition = {
+  // Common text components
+  shared: {
+    buttonStatus: {
+      success: () => `Success`,
+      error: () => `Error`,
+      retry: () => `Reintentar`,
+      sent: () => `Envidado`,
+      request: () => `Solicitar`,
+    },
+    buttonAction: {
+      confirm: () => `Confirmar`,
+      cancel: () => `Cancelar`,
+      decline: () => `Rechazar`,
+      request: () => `Solicitar`,
+      send: () => `Enviar`,
+      accept: () => `Aceptar`,
+      submit: () => `Enviar`, // TODO: see the context
+    },
+    textPressable: {
+      cancel: () => `Cancelar`,
+    },
+  },
+  // -------------------------- SCREENS --------------------------
+  // ------------ HISTORY ------------
+  // HistoryList.tsx
+  historyList: {
+    empty: () => `Ninguna transacción`,
+    screenHeader: {
+      default: () => `Actividad reciente`,
+      other: () => `Entre ustedes`,
```

Comment:
this was the last one, @Ethanol48 let me know if this looks right!

Reply:
Could you provide me with a screenshot when this comes up in the app? To see the context